lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.scss

```diff
@@ -1,12 +1,12 @@
-.adf {
-    &-permission-card {
-        height: 100%;
-        box-sizing: border-box;
-        display: flex !important;
-        flex-direction: column;
-        overflow: hidden;
-    }
+adf-permission-list .adf-permission-card {
```

Comment:
Any reason why the structure for the styles was changed? It doesn't have any functional impact, but I would suggest/prefer the older way of writing styles, with a little addition to ensure consistency with the rest of the codebase.
```
adf-permission-list {
    .adf {
        &-permission-card {
            ...
            ...
        }
        ...
        ...
    }
    ...
    ...
}

Reply:
this change is not required, removing it.

Reply:
Cool, thank you